presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,76 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries
```

Comment:
Fix ... error for queries that run in the reserved memory pool with spilling enabled.